app/src/test/java/org/simple/clinic/teleconsultlog/prescription/TeleconsultPrescriptionUpdateTest.kt

```diff
@@ -93,4 +95,28 @@ class TeleconsultPrescriptionUpdateTest {
             ))
         ))
   }
+
+  @Test
+  fun `when data for next click is loaded, then create prescription`() {
+    val medicalInstructions = "This is a medical instructions"
+    val medicalRegistrationId = "ABC12345"
+    val bitmap = mock<Bitmap>()
```

Comment:
Something to think about later, but we can remove the need for mocking a platform type by changing the event to have a boolean property (`hasSavedSignature`) which can be set in the effect handler. This will let us have the added benefit of making `.update()` pure again.

Reply:
Yeah, I initially wanted to use that way. But then it seemed like it was a logical decision that is being made in effect handler. But yeah, I can make this change in this PR.

Reply:
The logical decision here is _what_ to do if there is no saved signature, which is still being decided in the `update()` function. What we are changing is that the effect handler will tell the `update()` function whether there is a saved signature or not.